.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'bug'
+assignees: ''
+
+---
+
+### Describe the bug
+A clear and concise description of what the bug is.
+
+### System information (command line)
+ - OS: [e.g. iOS]
+ - Version [e.g. 3.0.0]
+ - Python version [e.g. Python3.9.1]
+ - Shell [e.g. ZSH]
+
+### System information (web service)
+ - OS: [e.g. iOS]
+ - Version [e.g. 3.0.0]
+ - Molgenis database version [e.g. 7.4.1-GAVIN2-1.0.0-SNAPSHOT]
```

Comment:
Molgenis database version?

Reply:
It's a literal copy-paste from https://github.com/molgenis/capice/tree/main/.github to ensure we have something here. I'd suggest either closing this PR and fixing it there first or merge and fix it later in both.